test/views/behaviours/ViewBehaviours.scala

```diff
@@ -75,21 +74,17 @@ trait ViewBehaviours extends ViewSpecBase {
 
         "have the correct banner title" in {
 
-          val doc = asDocument(view)
-          val bannerTitle = doc.getElementsByClass("hmrc-header__service-name hmrc-header__service-name--linked")
-          bannerTitle.html() mustBe messages("service.name")
+          findBannerTitle(view) mustBe messages("service.name")
         }
 
         "display the correct browser title" in {
 
-          val doc = asDocument(view)
-          assertEqualsMessage(doc, "title", s"$messageKeyPrefix.title", messageKeyParam)
+          findBannerTitle(view) mustBe messages("service.name")
```

Comment:
Not sure this is checking the correct element? The function is for the banner title